action.yml

```diff
@@ -21,11 +22,19 @@ inputs:
   # store this file in the .github folder in your repository
   profile_filename:
     description: "Name of the test configuration file"
+    required: false
 
   # name of the performance profile to use to run tests
   # see: https://docs.meshery.io/functionality/performance-management#performance-profiles
   profile_name:
     description: "Name of the performance profile"
+    required: false
+
+  # overrides the endpoint_url specified in the tes configuration file
+  # use this for dynamically injecting the application endpoint while running the workflow
+  endpoint_url:
+    description: "Endpoint in which the application is deployed"
+    required: false
 
 runs:
   using: "node12"
```

Comment:
What's "node12"?

Reply:
Node.js 12
The entry point of the action is a JS file.

Reply:
Ok, thank you for clarifying.